gamma/2.1/README.md

```diff
@@ -0,0 +1,66 @@
+# GAMMA Container
+
+## Documentation and Overview
+
+Main Tool: [GAMMA: a tool for the rapid identification, classification and annotation of translated gene matches from sequencing data](https://academic.oup.com/bioinformatics/article/38/2/546/6355578)
+
+Full documentation: [GAMMA Github](https://github.com/rastanton/GAMMA) 
+
+GAMMA (Gene Allele Mutation Microbial Assessment) is a command line tool that finds gene matches in microbial genomic data using protein coding (rather than nucleotide) identity, and then translates and annotates the match by providing the type (i.e., mutant, truncation, etc.) and a translated description (i.e., Y190S mutant, truncation at residue 110, etc.). Because microbial gene families often have multiple alleles and existing databases are rarely exhaustive, GAMMA is helpful in both identifying and explaining how unique alleles differ from their closest known matches.
+
```

Comment:
Could you include more information about the databases that are included in the container?

Reply:
@erinyoung, GAMMA doesn't actually need them to run. I can delete lines 42-56, 64-72 and it runs and builds fine. GAMMA really just needs Blat, biopython, @rastanton can correct me if I am wrong on this. Not sure why those files were included before, figured I would just keep them in case whoever ran v1.4 wants to upgrade and needs them.

Reply:
I think those databases might be useful. I'm just unsure if they are useful or really what's the usage for using them.

We should probably remove them if they're just taking up space.

Reply:
I just removed them and I can add them in again if its useful to others.

Reply:
@jvhagey that is correct, those files are not needed, thanks.